src/fides/cli/commands/view.py

```diff
@@ -1,8 +1,9 @@
 """Contains the view group of the commands for fides."""
 
 import click
+from toml import dumps as toml_dumps
 
-from fides.cli.utils import pretty_echo, with_analytics
+from fides.cli.utils import with_analytics, print_divider
```

Comment:
Nit: Should these be alphabetical?

Reply:
no idea how that slipped through and didn't trigger a CI failure....fixing